tests/series_only/__getitem___test.py

```diff
@@ -45,3 +50,14 @@ def test_index(constructor_eager: ConstructorEager) -> None:
     df = constructor_eager({"a": [0, 1, 2]})
     snw = nw.from_native(df, eager_only=True)["a"]
     assert snw[snw[0]] == 0
+
+
+@pytest.mark.filterwarnings(
+    "ignore:.*_array__ implementation doesn't accept a copy keyword.*:DeprecationWarning:modin"
+)
```

Comment:
is this just something that Modin calls (on itself?) internally? not anything we need to concern ourselves with?

Reply:
Yes, it traces back to [modin/pandas/indexing.py](https://github.com/modin-project/modin/blob/8c7799fdbbc2fb0543224160dd928215852b7757/modin/pandas/indexing.py#L1129-L1133)